src/package.xml

```diff
@@ -52,6 +52,10 @@
         <members>CON_Email_TEST</members>
         <members>CON_EthnicityRace_BATCH</members>
         <members>CON_EthnicityRace_TEST</members>
+        <members>CON_Phone</members>
+        <members>CON_Phone_Batch</members>
```

Comment:
This needs to be removed from the package.xml - I can't deploy to my org otherwise.